.github/workflows/luacheck.yml

```diff
@@ -63,3 +63,16 @@ jobs:
         with:
           report_paths: 'busted.xml'
           check_name: 'Test Report'
+
+  lua-type-check:
+    name: Type Check Code Base
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Type Check Code Base
+        uses: mrcjkb/lua-typecheck-action@v0.2.0
+        with:
+          checklevel: Error
```

Comment:
Looks like it is not checking at all since it is trying to check a directory "lua" which we do not have?
`lua-language-server --check "/home/runner/work/Lua-Modules/Lua-Modules/lua" --logpath=. --checklevel=Error`

```suggestion
          checklevel: Error
          directories: |
            ./
```